code/game/jobs/job/civilian_vr.dm

```diff
@@ -57,3 +57,39 @@
 /datum/job/chaplain
 	pto_type = PTO_CIVILIAN
 
+
+
+//////////////////////////////////
+//			Entertainer
+//////////////////////////////////
+
+/datum/job/Entertainer
```

Comment:
This specific path is only capitalized one

Reply:
What do you mean? As in I should make it say /datum/job/ENTERTAINER ?

Reply:
rather, most are /datum/job/lowercasename, so in this case it'd be /datum/job/entertainer

Reply:
As in it's the only one that starts with a capital letter. All other job paths are entirely lowercase.

Reply:
Fixed this.